doc/src/properties/atom.toml

```
@@ -0,0 +1,41 @@
+[is_hetatm]
+type = "bool"
+
+PDB = """When reading, **is_hetatm** is set to ``true`` for atoms defined with a
+``HETATM`` record, and ``false`` for atoms defined with an ``ATOM`` record.
+When writing, **is_hetatm** is used to determine whether to emit an ``HETATM``
+or an ``ATOM`` record. If the property is not set, ``HETATM`` is used."""
+
+MMTF = """When reading, **is_hetatm** is set to ``false`` when the
+``composition_type`` for the group is related to peptide or nucleotide
+linkage. See the ``composition_type`` property for residues for more
+information. This property is ignored while writing."""
+
+mmCIF = """When reading, **is_hetatm** is set to ``true`` when
+``_atom_site.group_PDB`` is ``HETATM``, and ``false`` when it is ``ATOM``. When
+writing, **is_hetatm** is used to determine whether to use ``HETATM`` or
+``ATOM`` for ``_atom_site.group_PDB``. If the property is not set, ``ATOM`` is
+used."""
+
+[altloc]
+type = "string"
+
+MMTF = """On reading, this property is set the the alternative location
+character stored in both of these formats. On writing, this character is stored
+with the ``ATOM`` or ``HETATM`` record. If the property is not set, a space
+character is used."""
+
+mmCIF = """On reading, this property is set the the alternative location
+character stored in both of these formats. On writing, this character is stored
+with the ``ATOM`` or ``HETATM`` record. If the property is not set, a space
+character is used."""
+
+[sybyl]
+type = "string"
+
+MOL2 = """The *sybyl* atom type typically stored in a MOL2 file. These types
```

This uses RestructuredText syntax

```suggestion
MOL2 = """The **sybyl** atom type typically stored in a MOL2 file. These types
```